File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.swift
func solution(_ n:Int, _ m:Int) -> [Int] {
    var result = [1,1]
    
    for i in 2...max(n,m) {
        if (n%i == 0 && m%i == 0) { result[0] = i }
    }
    
    for i in 1...m*n {
        if (i%n == 0 && i%m == 0) { result[1] = i; break; }
    }
    
    return result
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int[] answer = {0,0};
        for(int i=1; i<=n; i++) if(n%i == 0 && m%i == 0) answer[0] = i;
        for(int i=m; i<=n*m; i++) {
            if(i%n == 0 && i%m == 0) {
                answer[1] = i;
                break;
            }
        }
        return answer;
    }
}